core/src/datayoga_core/blocks/http/receiver/block.schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "http.receiver",
+  "description": "Receives HTTP requests and process the data.",
+  "type": "object",
+  "properties": {
+    "host": {
+      "description": "Host to listen",
+      "type": "string",
+      "default": "0.0.0.0"
+    },
+    "port": {
+      "description": "Port to listen",
+      "type": "integer",
+      "default": 8080
+    }
+  },
+  "additionalProperties": false,
+  "examples": [
+    {
+      "host": "localhost",
+      "port": 8080
+    }
+  ]
+}
```

Comment:
It seems to me that the following properties are missing:
- endpoint/uri: currently it's host:port
- https: boolean
- method: GET/POST, etc.
- Credentials for basic authentication.

Reply:
I don't think we have to support https nor auth here.
The same applies to the method: it doesn't matter which one is used - producer will accept everything have a body.

Why? It's really hard to implement properly all those thinks. 
I mean, It's not only https, it's also cert based auth, external auth, LDAP and so on.

Instead of reimplementing all of this on our side we just forcing users to use appropriate component(traefik, nginx, etc) on top of runner.